tst/standard/ClassicalNormalizerMatrixGroups.tst

```diff
@@ -1,56 +1,84 @@
 gap> n := 4;; q := 3;;
 gap> G := SymplecticNormalizerInSL(n, q);;
+gap> IsSubset(SL(n, q), GeneratorsOfGroup(G));
+true
 gap> Size(Group(GeneratorsOfGroup(G))) = Size(G);
 true
 gap> n := 4;; q := 5;;
 gap> G := SymplecticNormalizerInSL(n, q);;
+gap> IsSubset(SL(n, q), GeneratorsOfGroup(G));
+true
 gap> Size(Group(GeneratorsOfGroup(G))) = Size(G);
 true
 gap> n := 6;; q := 4;;
 gap> G := SymplecticNormalizerInSL(n, q);;
+gap> IsSubset(SL(n, q), GeneratorsOfGroup(G));
```

Comment:
This kind of testing doesn't scale too well. I would recommend to write a function `TestSymplecticNormalizerInSL := function(n,q) ... ` which performs all those tests and calls `error` with a meaningful message.

Then the rest of the `tst` file just becomes a bunch of nested loops calling this function with different values. And if you want to add a test in all cases, you only have to modify that one place.

Here is an example for this pattern in GAP: https://github.com/gap-system/gap/blob/master/tst/testinstall/grp/classic-forms.tst

Reply:
When I first read this, I thought this was an excellent idea! 

However, thinking about this a bit, it might not be as straightforward as I thought: Many of the functions include casework with divisibility conditions and sometimes, there are just no "small" examples for such cases - so to test these somehow, I decided to just check if the generators are in the SU (or SL, respectively), i.e. I removed the `Size(Group(GeneratorsOfGroup(G))) = Size(G)` line since that simply took too long to compute. 

Bottom line: It would be nice to be able to manually control for which values both checks (generators in SU and right size) are run and for which values only one of them (i.e. generators in SU) is used. 

I guess one could still make this work by having several testing functions, such as `TestSymplecticNormalizerInSLSize` and `TestSymplecticNormalizerInSLDeterminant`, and calling each of them only on the particular values for which one wants that particular test to be run. - Would you recommend that?

Reply:
I think the `Size` problem can and should be fixed by using `recog` to compute the size.

Reply:
This is handled in PR #36.